book/src/user_manual/credential_validation.md

```diff
@@ -2,10 +2,10 @@
 
 Credential validation is a process that allows a member to verify the validity
 of the credentials of other members in the group. The process is described in
-detail in the [MLS protocol specification](https://www.rfc-editor.org/rfc/rfc9420.html#name-credential-validation).
+detail in the [MLS protocol specification](https://messaginglayersecurity.rocks/mls-protocol/rfc9420.html#name-credential-validation).
```

Comment:
unclear if this is the right TDL, could point to https://datatracker.ietf.org/doc/html/rfc9420#name-key-packages or https://www.rfc-editor.org/info/rfc9420#name-key-packages